app/src/main/java/com.example.weatherapp/WeatherAdapter.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.weatherapp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.example.weatherapp.data.model.ForecastModel
+import com.example.weatherapp.databinding.WeatherCardItemBinding
+import com.example.weatherapp.utils.Constants
+
+class WeatherAdapter(private val days: List<ForecastModel>) :
+    RecyclerView.Adapter<WeatherAdapter.WeatherViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
+        WeatherViewHolder(
+            LayoutInflater.from(parent.context).inflate(R.layout.weather_card_item, parent, false)
```

Comment:
Fran, try to create an extension from ViewGroup so its easier to inflate

Reply:
Extension added